docs/usage.md

```diff
@@ -0,0 +1,98 @@
+
+```sh
+uvx repoauditor --include GitHub \
+  --GitHub-url https://github.com/<username>/python-helloworld \
+  --GitHub-pat ~/PAT.txt
+```
+
+`RepoAuditor` will generate a series of messages describing all the issues in the repository, along with the rationale behind them and the steps for resolution.
+
+<br/>
+
+## Custom Settings
+
+`RepoAuditor` tries to provide default settings and checks which are considered best practices. However, your team or organization may have different ways of doing things.
+You can override the default settings by specifying the appropriate flag.
+
+We have 3 types of flags:
+
+1. Boolean flags for those settings which are checked to be on by default and you wish to check are off, with the form `Module-no-Requirement` (e.g. `--GitHub-no-MergeCommit`).
+2. Conversely, boolean flags for those settings which are checked to be off by default, and you wish to check are on, denoted by the form `Module-Requirement` (e.g. `--GitHub-AllowDeletions`).
+3. Values flags take a string value to determine what value to enforce, and have the form `Module-Requirement-value` (e.g. `--GitHub-License-value "MIT License"`).
+
+Please run `uvx repoauditor --help` to get the list of all flags available.
+
+<br/>
+
+## Examples
+
+In this section, we will look at various exampls of using `RepoAuditor`. For the sake of brevity, we will omit the `--include` and `--GitHub-url` flags, leaving the inclusion to the user.
+
+`RepoAuditor` is designed to have sensible defaults which represent industry best practices.
+For example, `RepoAuditor` by default checks that the setting `Allow rebase merging` is off. However, if you wish to check that this setting is turned on, you can run:
+
+```sh
+uvx repoauditor --GitHub-RebaseMergeCommit
+```
+
+Similarly, `RepoAuditor` checks if `Issues` are enabled. You can set `RepoAuditor` to check if `Issues` is disabled by specifying
+
+```sh
+uvx repoauditor --GitHub-no-SupportIssues
+```
+
+Finally, `RepoAuditor` by default enforces that the `MIT License` exists for the repository. If your organization requires another license (such as `GPL`), you can enforce that by specifying the license keyword such as:
+
+```sh
+uvx repoauditor --GitHub-License-value "GNU General Public License v2.0"
+```
+
+The types of licenses supported by GitHub can be found [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#searching-github-by-license-type). Be sure to specify the full license name, e.g. `GNU General Public License v2.0`.
+
+<br/>
+
+## Excluding Requirements
+
+There is often the case you might wish to exclude a module or a specific requirement.
+`RepoAuditor` has built-in support for this as well, by using the `--excludes` flag.
+
+To exclude a requirement, you can run specify the requirement as `Module-Requirement`
+
+```sh
+uvx repoauditor --include GitHub --excludes GitHub-RequireSignedCommitsRule
```

Comment:
All instances of `--excludes` should be `--exclude`.

Reply:
Updated. Sorry about that, though I was expecting it to get resolved after we merged in #232.